### PR TITLE
Landlord immigration check: Add route for tenants from somewhere else

### PR DESCRIPTION
**Pull request by ikennaokpala**
[Trello card](https://trello.com/c/aSc4Ufh5/314-(8)-implement-from-other-countries%2Fsomewhere-else---landlord-immigration-check)

## Description 

Follows on from the changes in branch: [feature/landlord-immigration-check-flow-changes](https://github.com/alphagov/smart-answers/compare/feature/landlord-immigration-check-flow-changes)

This PR adds the From somewhere else option and route. 

## Factcheck
[Preview Link](https://smart-answers-pr-3061.herokuapp.com/landlord-immigration-check/y/B1%201PW/yes/yes)
[GOV.UK](https://www.gov.uk/landlord-immigration-check/y/B1%201PW/yes/yes)

## Expected changes
* Add the flow for tenants who are From somewhere else or you don't know.

### Before
screen shot 2017-03-13 at 17 26 08

### After
screen shot 2017-06-06 at 13 45 40

### CODE REVIEW

File: lib/smart_answer_flows/landlord-immigration-check/questions/what_nationality.govspeak.erb [hunk 1]
```diff
@@ -6,4 +6,5 @@
   "british-or-irish": "a British or Irish citizen",
   "eea": "from another EU country, the EEA or Switzerland",
   "non-eea": "a non-EEA family member of someone from the EU, EEA or Switzerland",
+  "somewhere-else": "From somewhere else or you don’t knows",
```
Comment (cbaines) [inline lib/smart_answer_flows/landlord-immigration-check/questions/what_nationality.govspeak.erb]: `knows` -> `know` would read better.

Comment (ikennaokpala) [inline lib/smart_answer_flows/landlord-immigration-check/questions/what_nationality.govspeak.erb]: Thanks for pointing this out I will make amends.

Comment (cbaines) [inline lib/smart_answer_flows/landlord-immigration-check/questions/what_nationality.govspeak.erb]: This text is included in the first commit as well, if you'd like to amend it there also.

Comment (cbaines) [inline lib/smart_answer_flows/landlord-immigration-check/questions/what_nationality.govspeak.erb]: First commit *message* that is.

**Pull request merged**